exp/topology/desiredstate/desired_state_test.go

```diff
@@ -806,26 +806,6 @@ func TestComputeControlPlaneVersion(t *testing.T) {
 					Build(),
 				expectedVersion: "v1.2.2",
 			},
-			{
-				// Control plane is considered scaling if controlplane.spec.replicas is not equal to any of
-				// controlplane.status.replicas, controlplane.status.readyReplicas, controlplane.status.updatedReplicas.
-				name:            "should return controlplane.spec.version if the control plane is scaling",
-				topologyVersion: "v1.2.3",
-				controlPlaneObj: builder.ControlPlane("test1", "cp1").
-					WithSpecFields(map[string]interface{}{
-						"spec.version":  "v1.2.2",
-						"spec.replicas": int64(2),
-					}).
-					WithStatusFields(map[string]interface{}{
-						"status.version":             "v1.2.2",
-						"status.replicas":            int64(1),
-						"status.updatedReplicas":     int64(1),
-						"status.readyReplicas":       int64(1),
-						"status.unavailableReplicas": int64(0),
-					}).
-					Build(),
-				expectedVersion: "v1.2.2",
-			},
```

Comment:
What about preserving this test but checking that the version is propagated when CP is scaling up?